openshift/rh-che.config.yaml

```diff
@@ -46,3 +46,6 @@ data:
   che-logs-sentry-dsn: 'NULL'
   sentry-stacktrace-app-packages: 'com.redhat,org.eclipse.che'
   sentry-environment: 'dev'
+  analytics.segment_write_key: 'rIZ5NDpUfqkTNdfsBxOBcJoLDT3TxftM' 
```

Comment:
Even if this is not a valid key I think it's better if you set it to the empty string:
```
analytics.segment_write_key: '' 
```

Reply:
fixed